spring-boot/admin/src/main/java/eu/coatrack/admin/service/OAuthUserDetailsService.java

```diff
@@ -56,19 +73,29 @@ private OAuth2User getLoggedInUser() {
     }
 
     public String getLoginNameFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("login");
+        return getLoggedInUser().getAttribute(attributeNameForLogin);
     }
 
     public String getNameFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("name");
+        return getLoggedInUser().getAttribute(attributeNameForName);
     }
 
     public String getCompanyFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("company");
+        if ((attributeNameForCompany).equals(KEYROCK_COMPANY_ATTRIBUTE)) {
+            return getFirstOrganizationFromKeyrock();
+        } else
+            return getLoggedInUser().getAttribute(attributeNameForCompany);
+    }
+
+    private String getFirstOrganizationFromKeyrock() {
```

Comment:
Does this work? I remember that we discussed that getting organizations does not yet work, in that case this code can probably be dropped?

Reply:
So as we discussed last time, it seems that Keyrock has a complete different concept regarding organization/company, then CoatRack, and I then explained you, that I need to do a lot of "manual" work to make it working, but the most probable case, is that the user needs to fill the company in the register form.
Based on this, I would drop these changes

Reply:
Yes, I would suggest to drop the code. If we find a reasonable way to get organization info, we could do that in a follow-up, but this is not important for now.

Reply:
Done